src/Bicep.LangServer/Snippets/Templates/res-cosmos-account.bicep

```diff
@@ -11,17 +11,16 @@ resource cosmosDbAccount 'Microsoft.DocumentDB/databaseAccounts@2019-12-12' = {
     }
     locations: [
       {
-        locationName: ${6:location1}
+        locationName: ${6:testLocation}

```

Comment:
Can we change it to location and not use -test in product code?

Reply:
Good call, updated to location